warrior/Framework/Utils/email_utils.py

```diff
@@ -165,14 +165,24 @@ def compose_send_email(exec_type, abs_filepath, logs_dir, results_dir, result,
                 (1) per_execution(default)
                 (2) first_failure
                 (3) every_failure
+        7. compress - specifies if the result has to be compressed and emailed.
+           Supported options: Yes/No(default)
```

Comment:
Remove this as there is no arg called compress

Reply:
Addressed the comments